zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -556,7 +556,7 @@ ascan.progress.chart.4xx = 4xx
 ascan.progress.chart.5xx = 5xx
 ascan.progress.chart.responses = Responses / second
 ascan.progress.chart.time = Time 
-ascan.progress.copyclipboard.button.label = Copy to Clipboard
+ascan.progress.copyclipboard.button.label = Copy
 ascan.progress.copyclipboard.button.tooltip = Copies the contents of the table to the clipboard
 ascan.progress.copyclipboard.error = Failed to copy the contents to the clipboard.
```

Comment:
Should drop from there too?

Reply:
Sure. I kinda thought these were more purposefully explanatory but it doesn't really add much, I think clipboard is still a safe assumption here.